vignettes/laminr.Rmd

```diff
@@ -126,10 +127,12 @@ Here, marker genes are calculated for each of the provided cell type labels usin
 # Create a Seurat object
 seurat <- SeuratObject::CreateSeuratObject(
   counts = as(Matrix::t(adata$X), "CsparseMatrix"),
-  meta.data = adata$obs,
+  meta.data = adata$obs
 )
 # Set cell identities to the provided cell type annotation
 SeuratObject::Idents(seurat) <- "cell_type"
+# Normalise the data
+seurat <- Seurat::NormalizeData(seurat)
```

Comment:
I thought that this let's peoples memory and storage explode; that's why I removed it. It'd be a bad experience if upload or compute etc. took long -- so keeping data small would be good.

If it's not actually an issue anymore, it's all good!

Reply:
Ignored in new PR.